cypress/support/step_definitions/dashboard/listitems/the_textarea_should_show_if_i_click_the_request_changes_radio_button.js

```diff
@@ -1,5 +0,0 @@
-And("The textarea should show if I click the Request changes radio button", () => {
-  cy.findByLabelText("Request changes")
-    .check();
-  cy.get("#message");
```

Comment:
this is also an easy way to spot if something isn't accessible, if you can _only_ select it via id. 

(cypress, react, etc) testing library, Kent C. Dodds (the author) has a recommended order for selectors to ensure better accessibility. https://testing-library.com/docs/queries/about#priority

if you use getByRole, or getByLabelText, they are using accessible selectors. a screen reader is likely able to pick these elements up. if you get annoying things like when it selects multiple elements and throws an error, then it is probably going to cause issues with someone using a SR too.